source/dub/dub.d

```diff
@@ -422,6 +422,29 @@ class Dub {
 		write(dst.toPrettyString());
 	}
 
+	private void listPaths(string methodName)(BuildPlatform platform, string config)
+	{
+		string[] importPaths;
+
+		__traits(getMember, m_project, methodName)(importPaths, platform, config);
+
+		import std.stdio;
+
+		foreach(path; importPaths) {
+			writeln(path);
```

Comment:
Fair point. I'll just essentially write the two functions out instead. They are quite small anyway.